qa-ibd.planx-pla.net/etlMapping.yaml

```diff
@@ -68,3 +68,20 @@ mappings:
           - name: year
             src: year
             fn: set
+  - name: qa-ibd_project
+    doc_type: project
```

Comment:
i think we should also add the program name, https://github.com/uc-cdis/cdis-manifest/blob/d3c3083491d7b448e6dfd49e6bbe14645b1a6331/chicagoland.pandemicresponsecommons.org/etlMapping.yaml#L377-L378